Remove Prometheus dependency from the Swarm chart (#328)
hivemq/helm-charts: examples/README.md

### [hivemq-operator](./hivemq-operator)

Examples for the legacy HiveMQ Operator chart

### [hivemq-swarm](./hivemq-swarm)

> Includes an example about how to use and install a Prometheus ServiceMonitor along with a custom Grafana dashboard through a ConfigMap for the HiveMQ Swarm chart

Reviewer: ```suggestion
Examples for the HiveMQ Swam chart, including examples for Prometheus and Grafana monitoring.
```

## Documentation

> See the [HiveMQ Kubernetes documentation](https://docs.hivemq.com/hivemq-platform-operator/introduction.html) for more detailed information.

Reviewer: Lets remove this last documentation section and instead add a documentation link to each section above.
